### ISSUE TITLE
Option to disable bedrock users

### ISSUE THREAD

**Issue by mekb-turtle** [Minehut/Meta#406]
### Feature request

**Checklist**
- [yes] I have read and agree to the README (found on https://github.com/Minehut/Meta)
- [si] This is not a bug report, support request or plugin request
- [ja] I have searched the Issue Tracker to make sure this is not a duplicate issue
- [oui] This issue does not contain any sensitive information 

**Platform**

bedrock

**Describe the feature**

Add an option (in appearance or settings tab) to not let bedrock users join

**How the feature is useful**

Not everyone wants to or knows how to install another plugin like Skript and disable bedrock users. Bedrock users have different advantages/disadvantages, such as being able to scaffold without being completely over the block.

---
**Use 👍 and 👎 reactions to indicate you agree or disagree with this suggestion! Feel free to discuss the idea in the comments.**

**Comment by PundaChen** [comment]
1 add skript
2 create skript that kicks everyone with * in their name
3 add this skript to ur file manager
4 restart and ur done

**Comment by mekb-turtle** [comment]
> 1 add skript
> 2 create skript that kicks everyone with * in their name
> 3 add this skript to ur file manager
> 4 restart and ur done

i put in the description, Not everyone wants to or knows how to install another plugin like Skript and disable bedrock users.

**Comment by DaRealMinecraftexpert** [comment]
How about the Reverse? Disable Java users?

**Comment by Blocksnmore** [comment]
> How about the Reverse? Disable Java users? (If Disabling Java Players was a thing, Maybe MH could take a Step closer to allowing Emotes)

Few reasons i feel it wouldn't happen.
1: A majority of the mh community uses java
2: bedrock is very buggy
3: the servers themselves are java based 
heres some skript code if you want to allow only bedrock
```vb
on connect:
    if name of player does not contain "*":
        kick player
```

**Comment by DaRealMinecraftexpert** [comment]
I ask, cuz I feel BR is underappreciated, so I think I want to make an exclusive server for them. Also, That code is Skript right?


> 
> Few reasons i feel it wouldn't happen.
> 1: A majority of the mh community uses java
> 2: bedrock is very buggy
> 3: the servers themselves are java based
> heres some skript code if you want to allow only bedrock
> 
> ```vbnet
> on connect:
>     if name of player does not contain "*":
>         kick player
> ```

**Comment by Blocksnmore** [comment]
Yes it is skript

**Comment by LukasmanMHdude** [comment]
`does not contain` 🤔

**Comment by not-3than** [comment]
`on connect:
    if name of player contains "*":
        kick player due to "This server is not compatible with Bedrock users."`

**Comment by not-3than** [comment]
> `does not contain` 🤔

They were responding to someone asking to only allow bedrock users.

**Comment by drtshock** [comment]
Yes this is something we want to do and it's a valid suggestion. While you can do it with skript, that's not an acceptable implementation for everyone.

**Comment by mekb-turtle** [comment]
> Yes this is something we want to do and it's a valid suggestion. While you can do it with skript, that's not an acceptable implementation for everyone.

finally someone who understands what im trying to say lol
